pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "trustpilot"
+version = "8.0.0"
+description = "trustpilot api client including cli tool"
+authors = [
+    "Johannes Valbjørn <jgv@trustpilot.com>",
+    "Thomas Thiebaud <tht@trustpilot.com>"
+]
+readme = "README.md"
+repository = "https://github.com/trustpilot/python-trustpilot"
+license = "MIT"
+keywords = ["trustpilot", "api", "client"]
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+click = "^7.1.1"
+requests = "^2.23.0"
+aiohttp = "^3.6.2"
+async_generator = "^1.10"
```

Comment:
https://github.com/python-trio/async_generator is only for 3.5, its forfree in 3.6

Reply:
lets pull out this logic:
https://github.com/trustpilot/python-trustpilot/blob/switch-to-poetry/trustpilot/async_client.py#L9